src/prefect/cli/_types.py

```diff
@@ -47,11 +47,22 @@ def add_typer(
         typer_instance: "PrefectTyper",
         *args,
         no_args_is_help: bool = True,
+        pluralization_string: str = None,
```

Comment:
Can we just implement this as `alias`? It seems nice to have it refer to its purpose rather than the content we are providing.

Reply:
Yeah I think that's a good idea, I was also thinking to allow it to be a collection so we can have multiple aliases for typos etc